src/app/home/home.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component } from '@angular/core';
+import { MatDialog } from "@angular/material/dialog";
+import { RoomCreationModalComponent } from "./room-creation-modal/room-creation-modal.component";
+
+const WIDTH = '300px';
+
```

Comment:
I don't know mat, but maybe this can be defined in `rem` as well?

Reply:
I thought of it but I haven't set the root width and how can do that?

Reply:
I did, but I wasn't sure what the value of the `rem` should I give cos I still don't know the value of width in the root HTML, I could also use `%,` anyway, I changed it but I have to read more about `rem `:)

Reply:
1 rem is equal to font size of the html element, so 10px in our case